impeller/core/host_buffer.h

```diff
@@ -116,14 +114,42 @@ class HostBuffer final : public std::enable_shared_from_this<HostBuffer>,
   void Reset();
 
   //----------------------------------------------------------------------------
-  /// @brief Returns the size of the HostBuffer in memory in bytes.
+  /// @brief Returns the capacity of the HostBuffer in memory in bytes.
   size_t GetSize() const;
```

Comment:
(Just a drive-by comment, but this name is weird for what it is. It looks like the functionality is the same as it was before, though.)

Reply:
Yeah, should be called capacity. I changed the doc comment so it was approximately accurate...